.editorconfig

```diff
@@ -44,6 +44,7 @@ dotnet_style_qualification_for_event = false:suggestion
 # Use language keywords instead of framework type names for type references
 dotnet_style_predefined_type_for_locals_parameters_members = true:suggestion
 dotnet_style_predefined_type_for_member_access = true:suggestion
+dotnet_style_require_accessibility_modifiers = omit_if_default:warning
```

Comment:
This was just added Jan 31 or VS 15.7, so no effect just yet. We are ahead of the curve here.